Update Image Builder docs with multiple architectures (#13115)
kyma-project/test-infra: cmd/image-builder/github-workflow-integration.md

- **Tags**: The tags to be applied to the image.
- **ExportTags**: Whether to export the tags.
- **EnvFile**: The environment variables file.

> - **BuildEngine**: The build engine to be used. Allowed values are `kaniko` and `buildx`. Default is `kaniko`.

Reviewer: ```suggestion
- **BuildEngine**: The build engine to be used. Allowed values are `kaniko` and `buildx`. The default is `kaniko`.
```

> - **Platforms**: The platforms to be built. This is a list of platforms in the format `os/arch`. Default is `linux/amd64` and `linux/arm64` for `buildx` BuildEngine.

Reviewer: ```suggestion
- **Platforms**: The platforms to be built. The parameter value is a list of platforms in the format `os/arch`. For `buildx` BuildEngine, the default is `linux/amd64` and `linux/arm64`.
```

Reviewer: Or:

> - **Platforms**: The platforms to be built. This is a list of platforms in the format `os/arch`. Default is `linux/amd64` and `linux/arm64` for `buildx` BuildEngine.

Reviewer: ```suggestion
- **Platforms**: A list of platforms to be built in the format `os/arch`. For `buildx` BuildEngine, the default is `linux/amd64` and `linux/arm64`.
```
